MetaMask/metamask-mobile
# feat(2796): add a feature flag specific to the settings, it shows and hide then permission management list item in the list of settings

## **Description**
Adds a feature flag that can be used to show and hide the menu item in settings specific to permissions.


## **Related issues**

Contributes to: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

1. set the feature flag `export MM_PERMISSIONS_SETTINGS_V1_ENABLED="1"`
2. start with `yarn watch:clean` go to settings, you should see a Permissions menu item
3. set `MM_PERMISSIONS_SETTINGS_V1_ENABLED=""` and restart `yarn watch:clean` and the menu item should now be hidden 

## **Screenshots/Recordings**



| Before  (feature flag ON)    | After  (feature flag OFF)    |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/becd76de-227f-4606-a4e1-398a3873a620"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/fbc8ab6a-257b-4baa-b2dc-579ce3f31642"> |



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
